vendor/build-logger/src/ldlogger-util.c

```diff
@@ -61,6 +61,11 @@ char* shellEscapeStr(const char* str_, char* buff_)
   char* out = buff_;
   int hasSpace = strchr(str_, ' ') != NULL;
 
+  if(hasSpace){
+    *out++ = '\\';
+    *out++ = '\"';
+  }
```

Comment:
I think your intention here was to prepend two characters. But you are rewriting the first two.